backend/alembic/versions/4bd138c7e55b_initial.py

```
@@ -0,0 +1,125 @@
+"""initial
+
+Revision ID: 4bd138c7e55b
+Revises: 
+Create Date: 2025-04-27 01:32:34.788571
+
+"""
+from typing import Sequence, Union
```

Ruff (0.8.2)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)